### PR TITLE
Upgrade ember to 5.4.0 version

### LINKED ISSUE #728: Update packages to latest version and ember and ember-cli to 5.4
Please refer to this to upgrade ember version https://cli.emberjs.com/release/basic-use/upgrading/

### PR DESCRIPTION
**Pull request by satyam73**
### What is the change?

In this PR I have upgraded ember and it's related packages. Ember is upgraded from 4.6.0 to 5.4.0

### Is Development Tested?

- [x] Yes
- [ ] No

### Before / After Change Screenshots
N/A

**Note :**
What to delete and what not to

- Keep the headings and answer the questions.
- Delete the examples
- Delete the description below headings `What is the change?` & `Before/After Screenshots`

### CODE REVIEW

File: app/styles/cards.module.css [hunk 1]
```diff
@@ -119,21 +119,21 @@
   text-align: justify;
 }
 
-@media (max-width: 1250px) {
+@media (width <= 1250px) {
```
[the remark this answers is not in the document]
Comment (satyam73) [reply app/styles/cards.module.css:122]: Yes we can use, try seeing it here https://developer.mozilla.org/en-US/docs/Web/CSS/@media#formal_syntax

File: .prettierignore [hunk 2]
```diff
@@ -1,25 +1,13 @@
...
 /blueprints/*/files/
-/vendor/
 
 # compiled output
 /dist/
-/tmp/
-
-# dependencies
-/bower_components/
-/node_modules/
```
[the remark this answers is not in the document]
Comment (satyam73) [reply .prettierignore:11]: yes I think it got removed when upgrading Ember, will add it

File: .eslintignore [hunk 3]
```diff
@@ -1,25 +1,13 @@
...
 /blueprints/*/files/
-/vendor/
 
 # compiled output
 /dist/
-/tmp/
-
-# dependencies
-/bower_components/
-/node_modules/
```
[the remark this answers is not in the document]
Comment (satyam73) [reply .eslintignore:11]: https://github.com/Real-Dev-Squad/website-www/pull/757#discussion_r1414071608

**Review by MehulKChaudhari — APPROVED** [review]

File: codemods.log
Comment (ajoykumardas12) [inline codemods.log:1]: Should this log file be gitignored?

**Review by prakashchoudhary07 — APPROVED** [review]

**Pull request merged**